README.md

```diff
@@ -1,44 +1,44 @@
+To set up your own project, you will need a GitHub account and be logged in. Simply click on the "use this template" button. Tell GitHub where you want to spin up your prototype and click "create a repository from template".
 
+## Spinning up your local environment
 
-## Spinning up you  local environment
+You will need a git client, Node.js, and Yarn installed on your local machine for this part of the tutorial.
 
-For this part of the tutorial you will need and git client, node.js, and yarn on your  local machine
-
-In order to start developing locally you need to clone your new repository to your local machine. Then open your favorite CLI and simply navigate to the folder containing your repository and start up gatsby or next.js in development mode with the following commands.
+To develop locally, clone your new repository to your local machine. Open the terminal, navigate to the folder containing your repository, and start up Gatsby or Next.js in development mode with the following commands.
 
 **Nextjs**
 
-```cli
+"`cli
 $ yarn install
 $ yarn dev
+
 ```
 
 **Gatsby**
 
-```cli
+"`cli
 $ npm install
 $ npm run develop
 ```
 
-Afther yarn is ready with setting up your dev environment you should be able to navigate to [http://localhost:3000/](http://localhost:3000/) on Next or to [http://localhost:8000/](http://localhost:8000/) for gatsby and see the same application as on
-[https://conductionnl.github.io/nl-design-skeleton-gatsby/](https://conductionnl.github.io/nl-design-skeleton-gatsby/).  Feel free to open up your project code and make some changes you should see the changes in your development environment isntantainusly.
+After Yarn has successfully set up your dev environment, navigate to [http://localhost:3000/](http://localhost:3000/) on Next.js or to [http://localhost:8000/](http://localhost:8000/) for Gatsby and see the same application as on
+[https://conductionnl.github.io/nl-design-skeleton-gatsby/](https://conductionnl.github.io/nl-design-skeleton-gatsby/). Feel free to open up your project code and make some changes. You should see the changes in your development environment instantaneously due to hot-reloading.
 
 ## Publishing your prototype to the internet (gatsby only)
 
-The gatsby version of the skeleton application has build in support for github pages, meaning that you can build your application into a static website and publish it as a github page. The skeleton repository comes with a build in github action for publishing itsself as a github page, meaning that you can have your prototype automatically published to the internet on a code push.
+The Gatsby version of the skeleton application has build-in support for GitHub pages, meaning you can build your application into a static website and publish it as a Github page. The skeleton repository comes with a build-in GitHub action for publishing itself as a GitHub page. You can have your prototype automatically published to the internet on a code push this way.
```

Comment:
built-in*

Misschien de laatste regel omschrijven naar iets als: "You're prototype will be automatically published to GitHub pages when pushing to the repository." o.i.d.